supernode/daemon/mgr/preheat_mgr.go

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright The Dragonfly Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package mgr
+
+import (
+	"context"
+
+	"github.com/dragonflyoss/Dragonfly/apis/types"
+)
+
+// PreheatTask stores the detailed preheat task information.
+type PreheatTask struct {
+	ID         string
+	URL        string
+	Type       string
+	Filter     string
+	Identifier string
+	Headers    map[string]string
+
+	ParentID   string
```

Comment:
What are the fields "ParentID" and "Children" for? And how about add comments here?